cluster-http/src/test/scala/akka/cluster/http/management/scaladsl/ClusterHttpManagementRoutesSpec.scala

```diff
@@ -400,6 +404,8 @@ class ClusterHttpManagementRoutesSpec
 
         import scala.concurrent.duration._
 
+        implicit val patience: PatienceConfig = PatienceConfig(timeout = Span(3, Seconds), interval = Span(50, Millis))
```

Comment:
The flaky test in #848 doesn't seem to do anything fancier than any other test in that Spec so I suspect this PatienceConfig is not the fix.